cmake/SSGCommon.cmake

```diff
@@ -914,6 +914,20 @@ macro(ssg_build_product PRODUCT)
             "
         )
     endif()
+    if(SSG_ANSIBLE_TASKS_ENABLED)
+        install(
+            CODE "
+            file(GLOB ROLE_FILES \"${CMAKE_BINARY_DIR}/${PRODUCT}/playbooks/*.yml\") \n
```

Comment:
In fact, these files are Ansible Playbooks and you also have used the misleading word "tasks" for it before so I think using the term "role" for them increases confusion.

Reply:
that's correct... it was a copy paste problem